README.md

```diff
@@ -53,68 +74,111 @@ OpenTelemetry is a big ecosystem and everything doesn't fit into the goals of th
 
 We encourage your contributions to improve `newrelic-opentelemetry-examples`!
 
-Generally, we want to focus on the
-[getting started guides](#getting-started-guides)
-which all have a strict uniform structure. We're open to additional examples
-being added which have a volunteer [codeowner](#codeowners).
+Trivial changes, wording changes, spelling/grammar
+corrections, etc. can be made directly via pull requests and do not require
+an associated issue.
 
-We're more selective about additions or substantial changes to
-[other examples](#other-examples).
+Keep in mind that when you submit your pull request, you'll need to sign the
+CLA via the click-through using CLA-Assistant. You only have to sign the CLA
+one time per project. If you have any questions, or to execute our corporate
+CLA (which is required if your contribution is on behalf of a company), drop us
+an email at
+opensource@newrelic.com.
 
-We use the following criteria to evaluate additions:
+This repository has a few areas of emphasis, each with their own criteria for
+additions:
 
-* Does the example demonstrate a very popular use case or recurring pain point?
-* Has someone has volunteered to be a [codeowner](#codeowners)?
-* Is there documentation - either in the readme or
-  [docs.newrelic.com](https://docs.newrelic.com/) - which describes how to use
-  the data produced by the example in New Relic?
-* Is there continuous integration (i.e.
-  [github action](.github/workflows/pull_request.yml)) ensuring that the
-  example code functions?
+* [OpenTelemetry APM monitoring criteria](#opentelemetry-apm-monitoring-criteria)
+* [OpenTelemetry infrastructure monitoring criteria](#opentelemetry-infrastructure-monitoring-criteria)
+* [OpenTelemetry other examples criteria](#opentelemetry-other-examples-criteria)
 
-If the answer is yes to all those questions, please follow this process:
+We do not accept examples demonstrating workflows which are not related to New
+Relic. These should be contributed upstream to the appropriate OpenTelemetry
+project.
 
-* Create an issue in this repo and describe the proposed example.
-* Follow the issue workflow and make sure the issue is accepted before creating
-  a PR. If an issue has not been accepted then do not create a PR since it will
-  be rejected.
```

Comment:
I would still like this part of the process emphasized. Maybe at the very top: An issue must be opened first before opening a PR for any new example.

I think we should create an issue template "Proposal for a new example"